.ci-builds/data-filter.json

```diff
@@ -9,17 +9,22 @@
   },
 // Test mixed feature filter and resource filter
 // Exlude translit data so we can run test for ICU-20673
+// Also test for "whitelist" versus "include"
   "featureFilters": {
     "misc": {
       "whitelist": ["supplementalData"]
     },
-    "translit": "exclude"
+    "translit": "exclude",
+    "curr_tree": {
+      "filterType": "locale",
+      "include": ["my"]
```

Comment:
"include" should work but may not be the best option; see the doc for discussion

Reply:
Changed to includelist per discussion